Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py

```diff
@@ -382,74 +189,102 @@ def get_company_details_command(client, args):
+            }
+            ratings.append(rating_dict)
+
+        rating_details = []
+        for rating_detail_key in res_json.get('rating_details', {}):
+            rating_detail = res_json.get('rating_details', {}).get(rating_detail_key, {})
+            rating_detail_dict = {
+                'name': rating_detail.get('name'),
+                'rating': rating_detail.get('rating'),
+                'percentile': rating_detail.get('percentile'),
+                'display_url': rating_detail.get('display_url')
+            }
+            rating_details.append(rating_detail_dict)
+
+        readable = {
+            'Company Info': company_info,
+            'Ratings': ratings,
+            'Rating Details': rating_details
+        }
+
     else:
         res_json = res.json()
-        generic_iam_context = OutputContext_Details(errorCode=res.status_code,
-                                                    errorMessage=res_json)
-
-    generic_iam_context_dt = f'{generic_iam_context.command}(val.id == obj.id && val.instanceName == obj.instanceName)'
-
-    outputs = {
-        generic_iam_context_dt: generic_iam_context.data
-    }
+        readable = {}
+        generic_iam_context = {
+            'errorCode': res.status_code,
+            'errorMessage': res_json
+        }
 
     readable_output = tableToMarkdown(name='Get Company Details:',
-                                      t=generic_iam_context.data,
-                                      headers=["errorCode", "errorMessage", "guid", "customId", "name", "description",
-                                               "ipv4Count", "peopleCount", "shortName", "industry", "industrySlug",
-                                               "subIndustry", "subIndustrySlug", "homePage", "primaryDomain", "type",
-                                               "displayURL", "ratingDetails", "ratings", "searchCount",
-                                               "subscriptionType", "sparkline", "subscriptionTypeKey",
-                                               "subscriptionEndDate", "bulkEmailSenderStatus", "serviceProvider",
-                                               "customerMonitoringCount", "availableUpgradeTypes", "hasCompanyTree",
-                                               "hasPreferredContact", "isBundle", "ratingIndustryMedian",
-                                               "primaryCompany", "permissions", "isPrimary", "securityGrade",
-                                               "inSpmPortfolio", "isMycompMysubsBundle", "companyFeatures"],
+                                      t=readable,
+                                      headers=["Company Info", "Ratings", "Rating Details"],
                                       removeNull=True
                                       )
-    return (
-        readable_output,
-        outputs,
-        generic_iam_context.data
-    )
+    return readable_output, generic_iam_context, res_json
```

Comment:
It is better to return the commandResults here, except it from the main method and return_result the CommandResults

Reply:
Unit test cases file was expecting Input in this format. So I have followed the same approach.